### DISCUSSION TITLE
TMT-DIA Sage search problem (using astral mass spectrometer)

### DISCUSSION THREAD

**Discussion by C-P-G-R** [lazear/sage#141] [Q&A]
How are you?
I am currently doing a Sage search for TMT data in DIA mode as shown in the title, but I am not getting any results ("wide_window": both true and false). When I search in DDA mode of Exploris480 with the same parameters, I get results, so the parameters seem to be fine. Can you please confirm why this is happening?
The mgf file and parameter file I used for the search can be found at https://www.dropbox.com/scl/fo/yg6ams3ng9gsmx5kfxnr6/ACj3g9g4EgPPn-1Kc0ICVU0?rlkey=u69gw7srzlis0mobc6my9h4hz&st=owm2jfrd&dl=0

There are 2 directories in that location,
Exploris480 contains the successful cases, and the Astral_TMT-DIA directory contains the files in question.
Note that we are using a code modification file to read the mgf file and there is also a corresponding file (sage.exe).

**Comment by lazear** [comment]
Sorry, I cannot debug any closed-source modifications you have made to Sage. If you find an issue with the open source codebase, please feel free to open an issue on the issue tracker.
